File: zhiyoufy-web/src/model/dto/dms/AggDateHistogramReq.ts
export interface AggDateHistogramReq {
  guid: string;
  startTime: string;
  endTime: string;
  interval: string;
  timeZone: string;
  format: string;
  projectName: string;
  environmentName: string;
  templateName: string;
  runTag: string;
}

File: zhiyoufy-server-java/zhiyoufy-server/src/main/java/com/example/zhiyoufy/server/domain/dto/elkagg/AggDateHistogramReq.java
package com.example.zhiyoufy.server.domain.dto.elkagg;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@JsonInclude(JsonInclude.Include.NON_NULL)
@ToString
public class AggDateHistogramReq {
    String guid;
    String startTime;
    String endTime;
    String interval;
    String timeZone;
    String format;
    String projectName;
    String environmentName;
    String templateName;
    String runTag;
}
